doc/site/core/sequence.markdown

```diff
@@ -18,6 +18,18 @@ If it returns `false`, stops iterating and returns `false`. Otherwise, returns
     [1, 2, 3].all {|n| n > 2} // False.
     [1, 2, 3].all {|n| n < 4} // True.
 
+### **any**(predicate)
+
+Tests whether any element in the sequence passes the `predicate`.
+
+Iterates over the sequence, passing each element to the function `predicate`.
+If it returns `true`, stops iterating and returns `true`. Otherwise, returns
```

Comment:
I think @gsmaverick's point is that the predicate doesn't have to return literal boolean `true`, but can return any truthy value. `all` is the same way. Maybe we should clarify the docs by saying something like, "a true value"?